doc/source/tutorial/tutorial_miscellaneous.ipyml

```diff
@@ -58,18 +58,29 @@ cells:
 
 - code: |
     # values below 10 millions are set to 10 millions
-    pop.clip(minval=10**7)
+    population.clip(minval=10**7)
 
 
 - code: |
     # values above 40 millions are set to 40 millions
-    pop.clip(maxval=4*10**7)
+    population.clip(maxval=4*10**7)
 
 
 - code: |
     # values below 10 millions are set to 10 millions and 
     # values above 40 millions are set to 40 millions
-    pop.clip(10**7, 4*10**7)
+    population.clip(10**7, 4*10**7)
+
+
+- code: |
+    # Using vectors to define the lower and upper bounds
+    lower_bound = sequence(population.time, initial=5_500_000, inc=50_000)
+    upper_bound = sequence(population.time, 41_000_000, inc=100_000)
+    
+    print(lower_bound, '\n')
```

Comment:
does this render correctly (as nicely as cell outputs) in the documentation? I fear it might be different...

Reply:
Yes, it does